src/App.js

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import Routes from './routes';
 import Footer from './components/Footer';
-
+import NavigationBar from './components/NavigationBar';
 import './App.scss';
 
 const App = () => {
   return (
     <Router>
+      <NavigationBar />
```

Comment:
What do you think about renaming this to `Header`? For me, it looks more natural to see a Header and Footer together

Reply:
Ya, I think that's good.